src/compiler/diagnosticMessages.json

```diff
@@ -1724,6 +1724,10 @@
         "category": "Error",
         "code": 2657
     },
+    "'{0}' cannot be used as a static member identifier.": {

```

Comment:
I think we need a lit bit more information here; preferably something that will point that classes constructors are inherently functions. something like:

`Duplicate identifier 'name'. Class constructor functions implicitly have a property with the same name`